plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/stock-indicator/constants.tsx

```diff
@@ -15,3 +15,5 @@ export const BLOCK_DESCRIPTION: string = __(
 	'Display product stock status.',
 	'woocommerce'
 );
+
+export const ALLOWED_PRODUCT_TYPES = [ 'simple', 'variation' ];
```

Comment:
I just wanted to highlight that, since we are mimicking the frontend behavior, we won't show the `stock indicator` when the product type is `variable`. We show the `stock indicator` for variations with `stock management` enabled.